src/styles/dictionary_modal.scss

```diff
@@ -1,16 +1,22 @@
 .sidenav-concept-btn {
     margin-top: 30%;
  }
- .modal-fade {
-     padding-top: 10%;
-     opacity: 10 !important;
-     background-color: rgba(0, 0, 0, 0.5);
- }
- .modal-content {
-     width: 150%;
-     margin-left: -36%;
-     margin-top: 20%  
- }
+.dictionary-modal-fade {
+    opacity: 10 !important;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+.dictionary-modal-fade form {
+   margin: 0px 3%;
+}
+.dictionary-custom-modal {
+    max-width: 750px;
+    margin-top: 270px;
```

Comment:
@emmabaye, what does this do?

Reply:
It sets the maximum width of the modal to `750px` and places it at a good position within the viewport so it can be completely seen by the user.

Reply:
@emmabaye, I meant the `margin-top: 270px`?

Reply:
The margin-top places the modal dialog box within the viewport. It brings the modal to a level below the top of of the browser window and counters an existing offset which causes a part of it to be hidden above the window.